atmosphere/um_env.yaml

```diff
@@ -0,0 +1,50 @@
+AINITIAL: ''
+ASTART: INPUT/pre-industrial.astart
+AUSCOM_CPL: 'true'
+DATAW: '{work_path}'
+FASTRUN: 'true'
+IDEALISE: ''
+PAREXE: parexe
+PRINT_STATUS: PrStatus_Oper
+RPSEED: ''
+RUNID: HI-24
+TYPE: NRUN
+UM_ATM_NPROCX: '16'
+UM_ATM_NPROCY: '12'
+UM_NAM_MAX_SECONDS: '300'
+UM_NPES: '192'
+UM_SECTOR_SIZE: '2048'
+UM_STDOUT_FILE: atm.fort6.pe
+VN: '7.3'
+
+# Config input paths
+UNIT01: ''
+UNIT02: prefix
+UNIT04: STASHC
+UNIT05: namelists
+UNIT07: PI-01.out2
+UNIT08: /dev/null
+UNIT09: CONTCNTL
+UNIT10: xhist
+UNIT11: ihist
+UNIT12: thist
+UNIT14: errflag
+UNIT15: ''
+UNIT58: ''
+UNIT22: INPUT/STASHmaster
```

Comment:
There doesn't seem to be a corresponding input line for this file?

Reply:
The `STASHmaster` directory is a subdirectory of ` - /g/data/vk83/configurations/inputs/access-esm1p5/share/atmosphere/stash/2020.05.19/` and is loaded in with this input.